File: 30 Days of Code/Day 8: Dictionaries and Maps.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
m = {}
n = int(input())
for _ in range(n):
    name, phone_number = input().split(" ")
    m[name] = phone_number

while True:
    try:
        q = input()
    except Exception:
        break

    num = m.get(q)
    if num:
        print(f"{q}={num}")
    else:
        print("Not found")

File: 30 Days of Code/Day 8: Dictionaries and Maps.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <map>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    map<string, int> m;
    int n;
    cin >> n;

    string s;
    int phoneNumber;

    for(int i = 0; i < n; i++) {
        cin >> s;
        cin >> phoneNumber;

        m[s] = phoneNumber;
    }
    string q;
    while (cin >> q) {
        if(m.find(q) != m.end())
            cout << q << "=" << m[q] << endl;
        else
            cout << "Not found" << endl;
    }

    return 0;
}
